deploy.sh

```diff
@@ -14,10 +13,19 @@ gitCommitSubject=$(git log -1 --pretty='%h %s')
 
 declare -A loggedin
 
+if [[ -z "$1" ]]; then
+  luaFiles=$(find . -type f -name '*.lua')
+else
+  luaFiles=$1
+fi
```

Comment:
```suggestion
if [[ -n "$1" ]]; then
  luaFiles=$1
else
  luaFiles=$(find . -type f -name '*.lua')
fi
```
To keep the same operator in the `if` as on https://github.com/Liquipedia/Lua-Modules/pull/1950/files#diff-30883d1df2fc62f59d66255a70dc05f60d3d4ef4d3957e6e3e38e62f6e471bd0R24
Makes it easier for everyone who doesn't know bash well